build/includes/sdk.mk

```diff
@@ -126,9 +126,10 @@ ensure-build-sdk-image:
 # SDK client test against it. Useful for test development
 run-sdk-conformance-local: TIMEOUT ?= 30
 run-sdk-conformance-local: TESTS ?= ready,allocate,setlabel,setannotation,gameserver,health,shutdown,watch,reserve
+run-sdk-conformance-local: FEATURE_GATES ?=
 run-sdk-conformance-local: ensure-agones-sdk-image
 	docker run -e "ADDRESS=" -p 9357:9357 -p 9358:9358 \
-	 -e "TEST=$(TESTS)" -e "TIMEOUT=$(TIMEOUT)" $(sidecar_tag)
+	 -e "TEST=$(TESTS)" -e "TIMEOUT=$(TIMEOUT)" -e "FEATURE_GATES=$(FEATURE_GATES)" $(sidecar_tag)
```

Comment:
oooh! I didn't even realise we had a local conformance test. Nice catch!

Do we want to do what we do in the other conformance tests with the new functionality -- i.e. have one test run with the feature gates all enabled, and one without?

Reply:
Yes, that might be reasonable. I have added unit tests with and w/o FeatureGates at least for this new feature.